client/src/assets/imgs/GitHub-Mark/.DS_Store:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]

```

Comment:
What are those files for??

Reply:
Looks like it was autogenerated somehow. They're not needed so I'll delete them.